libslide/src/diagnostics.rs

```diff
@@ -0,0 +1,46 @@
+use crate::common::Span;
+
+/// The kind of a slide diagnostic.
+pub enum DiagnosticKind {
+    Error,
+    Note,
+}
```

Comment:
Whats a note? a warning? I think its worth changing the naming here or some little clarification

Reply:
Good point, I'll add comments